### ISSUE TITLE
file_exists(): open_basedir restriction in effect

### ISSUE THREAD

**Issue by wioxjk** [Mowie/Mowie#3]
`[Mon Jan 22 23:20:43.057240 2018] [proxy_fcgi:error] [pid 19444:tid 139912013186816] [client 192.168.60.83:55154] AH01071: Got error 'PHP message: PHP Warning:  file_exists(): open_basedir restriction in effect. File(../SimplePages/backend/confirm.ini) is not within the allowed path(s): (/srv/www.linux.pizza/htdocs:/tmp) in /srv/www.linux.pizza/htdocs/apps/SimplePages/config.php on line 25\n', referer: http://linux.pizza
`
Even if all file is in within the allowed path, it still triggers the "open_basedir" restriction.

**Comment by kolaente** [comment]
Seems like it has something to do with using `__DIR__` when looking for the file. I'll make some more tests and let you know.

**Comment by kolaente** [comment]
Which url did you open when the error occured?

**Comment by kolaente** [comment]
It appears to be resolved, could you verify on your end?

**Comment by wioxjk** [comment]
`[Sat Feb 03 18:43:47.118898 2018] [proxy_fcgi:error] [pid 22166:tid 139911738861312] [client 192.168.60.83:52048] AH01071: Got error 'PHP message: PHP Warning:  require_once(../vendor/autoload.php): failed to open stream: No such file or directory in /srv/www.linux.pizza/htdocs/admin/install.php on line 10\nPHP message: PHP Fatal error:  require_once(): Failed opening required '../vendor/autoload.php' (include_path='.:/usr/share/php') in /srv/www.linux.pizza/htdocs/admin/install.php on line 10\n'`

Issue is still there, when downloading the master as a zip, and extracting it on the hosting.

**Comment by wioxjk** [comment]
The catalogue `/vendor` is not present in the master

**Comment by kolaente** [comment]
Yes, you need to install the libraries with [composer](https://getcomposer.org/).
